.github/workflows/it.yml

```diff
@@ -79,7 +81,9 @@ jobs:
             "registry": {
                 "registry_url": "localhost:5000",
                 "registry_namespace": "",
-                "registry_auth": "Y2h1YW5sYW5nLmdjdzoxcWF6MndzeA=="
+                "registry_auth": "Y2h1YW5sYW5nLmdjdzoxcWF6MndzeA==",
```

Comment:
Is this sensitive?

Reply:
We can trim it though it should point to any public registry.